tests/test_MIRI.py

```diff
@@ -126,16 +126,16 @@ def test_MIRI(capsys):
     # run assertions for S3
     meta.outputdir_raw = (f'data{os.sep}JWST-Sim{os.sep}MIRI{os.sep}'
                           f'Stage3{os.sep}')
-    name = pathdirectory(meta, 'S3', 1, ap=4, bg=10)
+    name = pathdirectory(meta, 'S3', 1, ap=8, bg=20)
```

Comment:
I think all these edits should be reverted after adjusting the filename scheme as I commented above.